documentation/architecture-decision-records/ADR0001-i18n_library.md

```diff
@@ -0,0 +1,126 @@
+* Support for 150+ languages.
+* Runs in the browser and Node.js.
+
+Developer Experience:
+
+The documentation is good https://github.com/yahoo/react-intl/wiki
+
+Support: 206 issues - 1 bug
+
+Licensing: Free to use under the Yahoo Inc. BSD license. See the LICENSE file for license text and copyright information.
+
+Size:
+
+Bundle Size
+47.6 kB Minified
+13kB Minified + GZipped
+
+https://bundlephobia.com/result?p=react-intl@2.8.0
+
+Examples:
+https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-react-app-with-react-intl
+https://medium.freecodecamp.org/internationalization-in-react-7264738274a0
+
+---
+
+2. [react-intl-universal](https://github.com/alibaba/react-intl-universal)
+
+Created and maintained by Alibaba.
+
+It is an improvement of `react intl` in that it adds functionality for supporting translations in Vanilla JS. `react-intl` only does this in React Components.
+
+Features:
+* It can do what we want i.e format numbers, times and dates
+* Can run both in the browser and in NodeJS so we will be covered when we finally work on adding SPA support
+* Supports fetching of config files from a remote source e.g S3
+* Seems like it was built for a specific reason; supporting translations in Vanilla JS. This doesn't apply to us, as we’ll be using React components, but it doesn't prevent us from using it.
+
+Support:
+* Quick responses on both issues and Pull requests
+* 14 issues, 6 PRs
+
+
+Developer experience:
+* Some parts of the docs aren’t grammatically correct. One will notice the errors but it’s not bad enough that one won’t be able to understand the docs.
+* Noticed that there are several issues that are asking for new features that are available on react-intl but not on react-intl-universal. Seems like they aren’t actively working on new features at the moment. If the current version suits our needs, we won’t have to worry about this.
+
+Size:
+
+Bundle Size
+75.4 kB Minified
+24.7 kB Minified + GZipped
+
+https://bundlephobia.com/result?p=react-intl-universal@1.15.5
+
+Licensing: BSD 3-Clause "New" or "Revised" License granted by Alibaba. [License page info.](https://github.com/alibaba/react-intl-universal/blob/master/LICENSE.md)
+
+---
+
+3. [fbt](https://github.com/facebookincubator/fbt)
+
+Created by Facebook.
+
+Features:
+* Organizing your source text for translation
+* Composing grammatically correct translatable UI
+* Eliminating verbose boilerplate for generating UI
+
+FBT works by transforming your `<fbt>` and `fbt(...)` constructs via Babel plugins. These plugins serve to extract strings from source and lookup translate payloads generated at build-time. FBT creates tables of all possible variations for the given fbt phrase and accesses this at runtime.
+
+Developer experience:
+
+* Confusing syntax
+* Not many resources found, other than the docs which are not very clear
+* https://facebookincubator.github.io/fbt/docs/api_intro
+
+Support: 18 issues - 1 bug. Low activity in community
```

Comment:
no Bundle Size?

Reply:
> Bundlephobia links added for react-intl and react-intl-universal. fbt is not installable by npm so that was skipped, but it was the worst in terms of documentation & support anyway.

^^

Reply:
True: https://github.com/facebookincubator/fbt/issues/34

For completion we should try installing it as per the fbt README just so that the record says it adds xMB, but it sounds like the decision has already been made 👍